plugins/source/github/docs/tables/github_repository_sboms.md

```diff
@@ -0,0 +1,26 @@
+# Table: github_repository_sboms
+
+This table shows data for Github Repository Sboms.
+
+The composite primary key for this table is (**org**, **repository_id**).
+
+## Relations
+
+This table depends on [github_repositories](github_repositories.md).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_id|`uuid`|
+|_cq_parent_id|`uuid`|
+|org (PK)|`utf8`|
```

Comment:
Looks like we're missing a PK that to identify this resource (`org` and `repository_id` are set when referencing those columns).

Would `spdxid` work or we need to include `spdx_version` too?

Reply:
Just pushed a change to fix the failing test - My bad, forgot to mock the DependencyGraphService.

I could be wrong, but I think the Org and the Repository ID are already part of the primary key? I don't think we would really want to add any more columns, there should only be 1 SBOM per repository.

Reply:
All repos will probaly have the same `spdxid` and `spdx_version`, so I'm not sure they're suitable candidates for the primary key. 
<img width="391" alt="image" src="https://github.com/cloudquery/cloudquery/assets/21217225/910f0c0e-a551-4e7a-9ef4-20594f8976fe">

Reply:
> I could be wrong, but I think the Org and the Repository ID are already part of the primary key? I don't think we would really want to add any more columns to the PK, there will only be 1 SBOM per repository.

Correct Org and the Repository ID. If there can only be one SBOM per repository we don't need to add additional columns. Going over the docs that seems to be the case